src/solc_0.7/common/BaseWithStorage/WithERC721BaseToken.sol

```diff
@@ -0,0 +1,429 @@
+/* solhint-disable func-order, code-complexity */
+pragma solidity 0.7.1;
+
+import "@openzeppelin/contracts/utils/Address.sol";
+import "../Interfaces/ERC721TokenReceiver.sol";
+import "../Interfaces/ERC721Events.sol";
+import "../BaseWithStorage/WithSuperOperators.sol";
+import "../BaseWithStorage/WithMetaTransactionReceiver.sol";
+import "../Interfaces/ERC721MandatoryTokenReceiver.sol";
+
+contract WithERC721BaseToken is ERC721Events, WithSuperOperators, WithMetaTransactionReceiver {
```

Comment:
Minor details but I am not a fan of the naming `WithERC721BaseToken`
maybe `WithERC721` but I think here `ERC721BaseToken` might be better

For Admin, for example, it made sense to rename because `AContract is Admin` did not feel right

Reply:
Sure, no problem. I'll switch the name back to `ERC721BaseToken`
I actually meant to ask you about the "With" prefix.